src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -27,17 +27,7 @@ public class StationAcceptanceTest {
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
-
+        ExtractableResponse<Response> response = this.createStation("강남역");
```

Comment:
```suggestion
        var response = this.createStation("강남역");
```

타입추론을 사용해보는 건 어떨까요? 타입 추론의 장점과, 타입 추론을 사용할 경우 주의점에 대해서 알아볼까요?

Reply:
제가 현업에서 코틀린을 사용하고 있는데
타입추론을 사용하니까 확실히 개발할때는 편한데 나중에 타입이 알고 싶을 때(변수 이름으로 타입이 바로 와닿지 않는경우)는 한번 들어가서 봐야하니까 좀 불편하더라고요

현재 피드백 주신 부분도 resposne긴 하지만 어떤 형식의 response인지는 알기가 힘들 거 같은데 이런 경우에는 변수 명을 더 자세하게 쓰는 걸 추천하시나요?

Reply:
저도 현업에서 코틀린을 사용하고 있습니다 :) 코틀린 개발자를 만나니 반갑네요.

저는 코틀린의 타입추론을 만나고나서 자바에서도 타입추론을 적극적으로 사용하고 있습니다. 그 이유는 지영님이 말씀해주신 것 처럼 타입추론을 통하여 변수 명을 더 자세히 적을 수가 있기 때문이였어요. 

즉, 코드에 대해 가독성을 더 챙겨보자는 의미로 적극적으로 사용했습니다. 😊